neurovault/apps/statmaps/forms.py

```diff
@@ -761,9 +761,9 @@ def update_ttl_urls(self):
                               fname + ')(\"\^\^xsd\:anyURI\ \;)')
 
         hdr, urlprefix, nifti, ftr = re.search(ttl_regx,ttl_content).groups()
-        base_url = 'http://neurovault.org'
-        replace_path = base_url + os.path.join(self.instance.collection.get_absolute_url(),
-                                    self.instance.name)+'/'
+        base_url = 'http://127.0.0.1'
+        base_dir = os.path.dirname(self.instance.nidmresultstatisticmap_set.first().file.url)
+        replace_path = "%s/%s/" %(base_url,base_dir)
```

Comment:
I cannot replicate the error this line is fixing. The files on the website seem to have correct urls (see for example: http://neurovault.org/collections/877/fsl_course_av.nidm.ttl)